### PR TITLE
Update ocp-metrics.md

### PR DESCRIPTION
**Pull request by Rajitha20H**

### CODE REVIEW

**Comment by Rajitha20H** [comment]
Updated ocp-metrics.md as per the issue https://github.ibm.com/katamari/dev-issue-tracking/issues/26271. @Fryguy, can you please merge the PR so that the document is updated.

File: integration_with_openshift_container_platform/_topics/ocp-metrics.md [hunk 1]
```diff
@@ -51,16 +51,18 @@ To create these accounts:
     **metrics-deployer** service account. Configure the
     **metrics-deployer** account to have **edit** permissions in the
     **openshift-infra** project:
-
-        $ oadm policy add-role-to-user \
-            edit system:serviceaccount:openshift-infra:metrics-deployer
-
+    ````
```
Comment (Fryguy) [inline integration_with_openshift_container_platform/_topics/ocp-metrics.md:54]: This should only be 3 backticks, not 4

File: integration_with_openshift_container_platform/_topics/ocp-metrics.md [hunk 2]
```diff
@@ -51,16 +51,18 @@ To create these accounts:
...
 6.  The **heapster** account will be automatically created in [Deploying the Metrics Components](#ocp-metrics-deploy). However, pre-emptively
     grant it **cluster-reader** permission to the **openshift-infra**
     project:
 
-        $ oadm policy add-cluster-role-to-user \
-            cluster-reader system:serviceaccount:openshift-infra:heapster
+    ```
+       oc adm policy add-cluster-role-to-user \
+    cluster-reader system:serviceaccount:openshift-infra:heapster
+    ```
```
Comment (Fryguy) [inline integration_with_openshift_container_platform/_topics/ocp-metrics.md:65]: why was the alignment changed from 

```
oc adm policy add-cluster-role-to-user \
    cluster-reader system:serviceaccount:openshift-infra:heapster
```

to

```
   oc adm policy add-cluster-role-to-user \
cluster-reader system:serviceaccount:openshift-infra:heapster
```

File: integration_with_openshift_container_platform/_topics/ocp-metrics.md (hunk 1 above)
Comment (Rajitha20H) [reply integration_with_openshift_container_platform/_topics/ocp-metrics.md]: I have changed into three ticks

File: integration_with_openshift_container_platform/_topics/ocp-metrics.md (hunk 2 above)
Comment (Rajitha20H) [reply integration_with_openshift_container_platform/_topics/ocp-metrics.md]: I have fixed the alignment.

**Comment by Fryguy** [comment]
Backported to `najdorf` in commit 562932d0316ceb1c79929f6d2e54e1eac1765480.

```text
commit 562932d0316ceb1c79929f6d2e54e1eac1765480
Author: Jason Frey <fryguy9@gmail.com>
Date:   Fri Apr 8 10:39:24 2022 -0400

    Merge pull request #1658 from Rajitha20H/origin/RSP_ocpmetrics.md
    
    Update ocp-metrics.md
    
    (cherry picked from commit 9ed65db7f7f90c11402403a3ff3db4f7eb867d0a)
```

**Pull request merged**